benchmarks/CMakeLists.txt

```diff
@@ -316,6 +316,43 @@ endforeach()
 add_subdirectory(transport)
 
 
+# unstructured benchmarks
+# ----------------
+
+if (GHEX_ENABLE_PARMETIS_BINDINGS)
```

Comment:
The comment below suggest that the variable name should be changed into `GHEX_UNSTRUCTURED_MT` or something?

Reply:
I see, I changed the comment a bit. In the following foreach, CPU benchmarks support multithreading and the actual number of threads is controlled by an environment variable (single-thread is also possible), whereas the GPU one is single-threaded. I would say the common thing among this set of benchmarks is that they all need ParMeTis bindings, whether threading mode may vary.